src/components/LoginDialog.jsx

```diff
@@ -10,14 +10,18 @@ import PropTypes from 'prop-types';
 
 const uiConfig = {
   signInFlow: 'popup',
+  signInSuccessWithAuthResult() {
+    return true;
+  },
```

Comment:
Ok removed it there since it already is doing that in the callbacks `property`
```javascript
callbacks: {
    signInSuccessWithAuthResult: () => true,
  },
```